irods/test/scripts/test002.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
```

Comment:
Is this comment acting as a hint for the reader to look at login_auth_test for details about test setup?

Reply:
No, but rather it has to do with tests are  wrapped to provide support for setup (and possibly teardown) surrounding the run of the test script itself. in #508 each test script has a designated wrapper script, [configured here](https://github.com/irods/python-irodsclient/blob/8cc6ee48bd696884dfa33f9405441675f1520032/irods/test/harness/test_script_parameters#L16) so there will be an equivalent entry for test002.bats and test003.bats. (which incidentally need to be given more fleshed-out names)

Reply:
Ok. We can save discussion of this for when work on #508 picks back up.

Carry on.